corehq/ex-submodules/pillowtop/migrations/0007_copy_xforms_checkpoint.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+from corehq.util.django_migrations import skip_on_fresh_install
+from pillowtop.models import KafkaCheckpoint
+
+
+@skip_on_fresh_install
+def copy_xforms_checkpoint(*args, **kwargs):
```

Comment:
It might be nice to put this in a utils file, I can imagine this being useful in the future